sdk/storage/Azure.Storage.DataMovement/src/TransferCheckpointStoreOptions.cs

```diff
@@ -10,25 +10,41 @@ namespace Azure.Storage.DataMovement
     public class TransferCheckpointStoreOptions
     {
         /// <summary>
-        /// The local folder where the checkpoint information will be stored.
+        /// Whether checkpointing should be enabled or not.
+        /// </summary>
+        public bool Enabled { get; private set; }
+
+        /// <summary>
+        /// The folder where the checkpoint information will be stored.
         /// </summary>
         public string CheckpointerPath { get; private set; }
```

Comment:
Does this need to be public for the other sub DataMovement libraries?

Reply:
These did not need to be public for any reason. I changed them to `internal`, hoping that is what you were suggesting.